playbooks/02 - Use Case - File Content Extraction/> Hunt Indicators on Qradar.json

```diff
@@ -0,0 +1,356 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "> Hunt Indicators on Qradar",
+    "aliasName": null,
+    "tag": null,
+    "description": "Hunt indicator value on FortiSIEM for the past X Minutes",
+    "isActive": true,
+    "debug": true,
```

Comment:
Best practice: debug should be false

Reply:
IMHO during QA/testing debug is of a great help to identify issues. Getting close to release this should be set to false

Reply:
Sounds good.